### DISCUSSION TITLE
Private server to GitHub connectivity

### DISCUSSION THREAD

**Discussion by vikramthippeswamy54** [community/community#130844] [Pull Requests]
### Select Topic Area

General

### GitHub Feature Area

Pull Requests

### Body

Hello Guys,

We have a private server which has no connectivity to internet and we have the code repo in GitHub. I Just wanted to pull the code from GitHub to private server. What is the safe and secure way ? Please suggest the best possible way out.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @vikramthippeswamy54 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
